tests/unittests/utils/test_radio.py

```
@@ -0,0 +1,1219 @@
+import numpy as np
+from PIL import Image
+import torch
+import traceback
+import tempfile
+import os
+import sys
+
+PASSED = 0
+FAILED = 0
+
+def _test(name):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_test`

(ANN202)